lib/internal/crypto/keys.js

```diff
@@ -76,12 +76,18 @@ class SecretKeyObject extends KeyObject {
 }
 
 const kAsymmetricKeyType = Symbol('kAsymmetricKeyType');
+const kParams = Symbol('kParams');
 
 class AsymmetricKeyObject extends KeyObject {
   get asymmetricKeyType() {
     return this[kAsymmetricKeyType] ||
            (this[kAsymmetricKeyType] = this[kHandle].getAsymmetricKeyType());
   }
+
+  get params() {
+    return this[kParams] ||
```

Comment:
Nit, but could you rewrite this as:

```js
if (this[kParams] === undefined)
  this[kParams] = this[kHandle].getAsymmetricParams();
return this[kParams];
```

Reply:
I was going to suggest something similar at first but it's locally consistent, `get asymmetricKeyType()` uses the same style.